src/core/system_impl.h

```diff
@@ -236,6 +236,13 @@ class SystemImpl : public Sender {
     SystemImpl(const SystemImpl&) = delete;
     const SystemImpl& operator=(const SystemImpl&) = delete;
 
+    void register_mavlink_command_handler(
+        uint16_t cmd_id,
+        MavlinkCommandReceiver::mavlink_command_handler_t callback,
+        const void* cookie);
+    void unregister_mavlink_command_handler(uint16_t cmd_id, const void* cookie);
+    void unregister_all_mavlink_command_handlers(const void* cookie);
```

Comment:
As I understand it, we are not using this interface just yet but in the future there will be a new plugin making use of it, and that one will be tested. Right?

Reply:
Yes that is correct